File: Cpp/LW3/lr3 z1/lr3 z1/abiturient.cpp
#include "abiturient.h"
#include <iostream>
using namespace std;

abiturient::abiturient(string pib, string date, string city, string phone, string specialty)
    : persona(pib, date, city, phone), specialty(specialty) {
}

void abiturient::info() const {
    cout << "Абiтурiєнт: " << pib << endl;
    cout << "Дата народження: " << date << " (Вiк: " << age() << ")" << endl;
    cout << "Заклад: " << city << endl;
    cout << "Телефон: " << phone << endl;
    cout << "Спецiальнiсть: " << specialty << endl;
    cout << "-----------------------------\n";
}

File: Csharp/LW8/lr8_1dll/lr8_1dll/abiturient.cs
using System;
using System.Security.Policy;

namespace lr8_1dll
{
    public class Abiturient : Persona
    {
        public string Specialty { get; set; }

        public Abiturient(string pib, string date, string city, string phone,
                          string specialty)
            : base(pib, date, city, phone)
        {
            Specialty = specialty;
        }

        public override void Info()
        {
            Console.WriteLine("Абітурієнт:");
            Console.WriteLine($"ПІБ: {PIB}");
            Console.WriteLine($"Спеціальність: {Specialty}");
            Console.WriteLine($"Місто: {City}");
            Console.WriteLine($"Телефон: {Phone}");
        }
    }
}
